### PR TITLE
Add category, required and class to form fields

### PR DESCRIPTION
**Pull request by DanielKuhn**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,3 +16,4 @@ spec/reports
 test/tmp
 test/version_tmp
 tmp
+.idea/
```
Comment (robsliwi) [inline .gitignore:19]: :rage:

File: lib/oat/adapters/siren.rb [hunk 2]
```diff
@@ -94,7 +94,7 @@ def initialize(name)
             @data = { :name => name }
           end
 
-          %w(type value title).each do |attribute|
+          %w(category required class type value title).each do |attribute|
```
Comment (robsliwi) [inline lib/oat/adapters/siren.rb:97]: Not alphabetic, but still fine :+1:

**Review by robsliwi — APPROVED** [review]

File: lib/oat/adapters/siren.rb (hunk 2 above)
Comment (DanielKuhn) [reply lib/oat/adapters/siren.rb:97]: wasn't alphabetical before either...

Comment (robsliwi) [reply lib/oat/adapters/siren.rb:97]: Because of this I accepted the additions :grin:

**Pull request merged**